index.html

```diff
@@ -3,6 +3,7 @@
 <head>
   <!-- Global site tag (gtag.js) - Google Analytics -->
   <script async src="https://www.googletagmanager.com/gtag/js?id=UA-148502409-1"></script>
+  <script type="text/javascript" src="http://code.jquery.com/jquery-latest.min.js"></script>
```

Comment:
Hey, what is the reason behind `jQuery` here?

Reply:
I was test some code but i will remove it.